src/Bound/TH.hs

```diff
@@ -141,61 +134,6 @@ makeBound' name vars cons = do
       bind :: ExpQ
       bind = constructBind name vars cons
 
-#if __GLASGOW_HASKELL__ < 708
-      def :: Name -> DecQ -> [DecQ]
-#if __GLASGOW_HASKELL__ < 706
-      def _theName dec = [dec]
-#else
-      def theName  dec = [pragInlD theName Inline FunLike AllPhases, dec]
-#endif
-
-      pureBody :: Name -> [DecQ]
```

Comment:
FWIW, this block is probably something which would allow us to use `TemplateHaskellQuotes`, if adapted to GHC-8+; instead of a splicing below.

Reply:
In fact, GHC-9 can splice inside quotes with just `TemplateHaskellQuotes`.

Reply:
I'll do this in a separate PR that addresses #85.